assignment/impl/src/java/org/sakaiproject/assignment/impl/GradeSheetExporter.java

```diff
@@ -70,6 +74,27 @@
     @Setter private FormattedText formattedText;
 
     private ResourceLoader rb = new ResourceLoader("assignment");
+	
+	/**
+	 * A comparator that sorts by student sortName
+	 */
+	static final Comparator<Submitter> SUBMITTER_NAME_COMPARATOR = new Comparator<Submitter>() {
```

Comment:
@joaquinmarques we have this identical comparator that you can use org.sakaiproject.util.comparator.UserSortNameComparator

Reply:
@ern I can't use directly org.sakaiproject.util.comparator.UserSortNameComparator because this comparator is for User class and my class is Submitter, so I can't do this:

submitters.sort(new UserSortNameComparator());

The error:

The method sort(Comparator<? super GradeSheetExporter.Submitter>) in the type List<GradeSheetExporter.Submitter> is not applicable for the arguments (UserSortNameComparator)

What would be the correct way to do this?